mypy.ini

```diff
@@ -0,0 +1,21 @@
+[mypy]
+python_version = 3.6
+files = Lib/fontTools/misc/plistlib
+follow_imports = silent
+ignore_missing_imports = True
+warn_redundant_casts = True
+warn_unused_configs = True
+warn_unused_ignores = True
+
+[mypy-fontTools.misc.plistlib]
```

Comment:
do we have to specify these options for each new module we add typing to? Are these specific to `plistlib` or can they be applied more generally?

Reply:
I did this in the spirit of gradual typing. plistlib is now a maximum security prison, but the rest of fontTools isn't. The plan is to add this stuff per-module until everything is typed, at which point we can switch to general settings.